src/modules/ledger/modules/chaincode-service-gateway/dto/contract.dto.ts

```diff
@@ -0,0 +1,55 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsArray, IsNotEmpty, IsString, IsUUID } from 'class-validator';
+
+export class CallContractInputDto {
```

Comment:
Why do we have two `CallContractInputDto` classes? The other one is [here](https://github.com/CyFy-Blockchain/fabric-gateway/pull/20/files#diff-698e69951f42c3904628e76f49c15adefca619cf401ddb945a018ac1d579965dR4).

Reply:
it was moved to make the module cleaner